### PR TITLE
Apiversion negotiation

### PR DESCRIPTION
**Pull request by tiborvass**
Follow up to https://github.com/moby/moby/pull/39032

Ping @thaJeztah @tonistiigi @vdemeester @AkihiroSuda

### CODE REVIEW

File: cli/context/docker/load.go [hunk 1]
```diff
@@ -127,6 +127,8 @@ func (c *Endpoint) ClientOpts() ([]client.Opt, error) {
 	version := os.Getenv("DOCKER_API_VERSION")
 	if version != "" {
 		result = append(result, client.WithVersion(version))
```
Comment (thaJeztah) [inline cli/context/docker/load.go]: WithVersion should now ignore empty values, so just adding both WithVersion and WithAPIVersionNegotiation should also work

**Comment by andrewhsu** [comment]
SGTM

**Comment by thaJeztah** [comment]
ping @silvin-lubecki

**Comment by thaJeztah** [comment]
oh; needs a rebase 😔

**Comment by thaJeztah** [comment]
rebased; removed the first commit, because master was up-to-date with those changes

**Pull request merged**